crypto.pl

```diff
@@ -33,7 +33,12 @@
 */
 
 :- module(crypto,
-          [ evp_decrypt/6,              % +CipherText, +Algorithm, +Key, +IV, -PlainText, +Options
+          [ crypto_term_hash/3,         % +Data, -Hash, +Options
```

Comment:
Code-lists are the least elegant way to represent strings, and they are *also* supported. Lists of characters are a good practical choice and *portable* with `double_quotes` set to `chars`. I note *portable* explicitly because *strings* are&nbsp;*not*.

"Data" is used in the documentation of these arguments already in `library(md5)` and `library(sha)`. "Data" never refers to a stream or file handle in the existing documentation.